src/redux/favouriteRedux.js

```diff
@@ -0,0 +1,28 @@
+/* selectors */
```

Comment:
Funkcjonalność dodawania produktów do ulubionych przeważnie jest bardzo prosta na stronach, jest ulubiony albo nie, w związku z tym nie ma potrzeby aby tworzyć całkowicie osobną logikę i domenę na potrzeby tej jeden funkcjonalności. W efekcie info o tym czy produkt jest ulubiony czy nie jest zwracane wraz z produktami, w postaci `boolean`. 

W naszym przypadku nie widzę powodu aby sobie to komplikować i trzymać favorite w całkowicie osobnym obiekcie stora, bo nie daje nam to żadnych dodatkowych plusów a tylko komplikuje cały kod.

Proponuje dodać po prostu pole `isFavourite: true | false` do produktów ;)